nautobot/extras/tables.py

```diff
@@ -60,6 +60,16 @@
 <button data-url="{% url 'extras:gitrepository_sync' slug=record.slug %}" type="submit" class="btn btn-primary btn-xs sync-repository" title="Sync" {% if not perms.extras.change_gitrepository %}disabled="disabled"{% endif %}><i class="mdi mdi-source-branch-sync" aria-hidden="true"></i></button>
 """
 
+JOB_RESULT_JOB = """
+{% if record.related_object and record.related_object.Meta and record.related_object.Meta.name %}
```

Comment:
Since we’re in a template her and invalid access won’t throw an error (I think? Might need to test that), maybe `{% if record.related_object.Meta.name %}` might be enough her?

Reply:
Good thought! I'll give it a try.